packages/astro/src/core/messages.ts

```diff
@@ -23,8 +23,18 @@ export function reload({ url, reqTime }: { url: string; reqTime: number }): stri
 }
 
 /** Display dev server host and startup time */
-export function devStart({ startupTime }: { startupTime: number }): string {
-	return `${pad(`Server started`, 44)} ${dim(`${Math.round(startupTime)}ms`)}`;
+export function devStart({ startupTime, port, localAddress, networkAddress, https, site }: { startupTime: number; port: number; localAddress: string; networkAddress: string; https: boolean; site: URL | undefined }): string {
+	const rootPath = site ? site.pathname : '/';
+	const toDisplayUrl = (hostname: string) => `${https ? 'https' : 'http'}://${hostname}:${port}${rootPath}`
+	const messages = [
+		``,
+		`${emoji('🚀', ' ')} ${magenta('astro v0.23.1')} ${dim(`started in ${Math.round(startupTime)}ms`)}`,
```

Comment:
We accidentally left this hardcoded, should pull in the actual version from `package.json`

Reply:
Fixed!